web/docs/project/env-vars.md

```diff
@@ -85,8 +85,8 @@ In the root of your Wasp project you can create two distinct files:
     REACT_APP_SOME_VAR_NAME=somevalue
     ```
 
-`.env.server` should not be committed to version control as it can contain secrets and is therefore by default already ignored in the `.gitignore` file that comes with a new Wasp app.
-On the other hand, `.env.client` can be committed as it must not contain any secrets.
+`.env.server` should not be committed to version control as it can contain secrets, while `.env.server` can be versioned as it must not contain any secrets.
```

Comment:
```suggestion
`.env.server` should not be committed to version control as it can contain secrets, while `.env.client` can be versioned as it must not contain any secrets.
```